src/yunohost/data_migrations/0021_migrate_to_bullseye.py

```diff
@@ -30,6 +30,59 @@
 N_NEXT_DEBAN = 11
 N_NEXT_YUNOHOST = 11
 
+VENV_BACKUP_SUFFIX = "_BACKUP_VENV"
+VENV_REQUIREMENTS_SUFFIX = "_req.txt"
+VENV_IGNORE = "ynh_migration_no_regen"
+
+def _get_all_venvs(dir,level=0,maxlevel=2):
+    result = []
+    for file in os.listdir(dir):
+        path = os.path.join(dir,file)
+        if os.path.isdir(path):
+            if os.path.isfile(os.path.join(path,VENV_IGNORE)):
+                continue
+            activatepath = os.path.join(path,"bin","activate")
+            if os.path.isfile(activatepath):
+                content = read_file(activatepath)
+                if "VIRTUAL_ENV" and "PYTHONHOME" in content:
```

Comment:
```suggestion
                if "VIRTUAL_ENV" and "PYTHONHOME" in content:
```
bool("VIRTUAL_ENV") -> True, so if content is equals to "PYTHONHOME" this condition is true

Reply:
Fixed in https://github.com/YunoHost/yunohost/pull/1479/commits/cdd579080833bc77e78246f78dbb16762c91fe14